socorro/external/es/base.py

```diff
@@ -4,6 +4,88 @@
 
 import datetime
 
+from configman import Namespace, class_converter
+
+from socorro.app.generic_app import App
+
+
+def string_to_list(input_str):
+    return [x.strip() for x in input_str.split(',') if x.strip()]
+
+
+class ElasticsearchConfig(App):
+
+    required_config = Namespace()
+    required_config.namespace('elasticsearch')
+    required_config.elasticsearch.add_option(
+        'elasticsearch_class',
+        doc='a class that implements the ES connection object',
+        default='socorro.external.es.connection_context.ConnectionContext',
+        from_string_converter=class_converter
+    )
```

Comment:
All options listed here: https://github.com/mozilla/socorro/blob/master/socorro/external/es/connection_context.py#L43 will automatically be added to this namespace by configman. You can thus avoid repeating them here. It means you have to open that file to see all the options, but it also mean less repeated code, which I believe is better.